metron-deployment/roles/java_jdk/defaults/main.yml

```diff
@@ -15,4 +15,4 @@
 #  limitations under the License.
 #
 ---
```

Comment:
I used Ambari's default so I don't have to worry about any introduced non-compatibility.